backend/inspirehep/files/ext.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import boto3
+from botocore.exceptions import UnknownServiceError
+from flask import _app_ctx_stack as stack
+from flask import current_app
+
+
+class InspireS3(object):
```

Comment:
why do we just call it `S3Extention` ?

Reply:
Because all the other extensions are called InspireSomething, so I think we should keep it consistent